src/libs/AxelarAssetTransfer.ts

```diff
@@ -266,6 +294,104 @@ export class AxelarAssetTransfer {
     return address.toLowerCase();
   }
 
+  /**
+   * @param {Object}  requestParams
+   * @param {string}  requestParams.fromChain - Source chain identifier eg: avalanche, moonbeam ethereum-2, terra-2 ...
+   * @param {string}  requestParams.toChain - Destination chain identifier eg: avalanche, moonbeam ethereum-2, terra-2 ...
+   * @param {string}  requestParams.destinationAddress - Address where the asset should be transferred to on the destination chain
+   * @param {Object}  requestParams.options
+   */
+  public async sendToken(requestParams: SendTokenParams) {
```

Comment:
This method is a bit too OP. Perhaps it makes sense to split this into two methods, one that allows the caller to sign the full tx and broadcast, and one that just returns the calldata/memo that needs to be passed in (similar to deposit address generation method letting the caller to handle the signing part). I imagine the first method will be useful for Satellite, but the second one would likely be what most apps want?

Reply:
To clarify: 
1. one method that signs and broadcasts, and 
3. another method that generates an unsigned tx for the user to sign and broadcast themself?

Reply:
What does "OP" mean?

Reply:
overpowered :p

yeah,

1. method to sign and broadcast
2. another method that just generates the IBC memo field to be used for send token. For EVM, it generates the tx calldata. In both cases, it also returns the contract address/IBC recipient address the tx is going too. This gives the caller flexibility to set gas params etc. itself, and also sign/broadcast
3. perhaps a third method that generates the unsigned tx using the IBC memo etc.

Reply:
cool, yeah this is exactly what i ended up doing, (without realizing what OP meant ;-) ).